### ISSUE TITLE
Why transform to 224 x 224

### ISSUE THREAD

**Issue by pjfalbe** [EntropyOrg/perl-AI-TensorFlow-Libtensorflow#19]
In the tutorial you transform the images to 224 x 224 why is that?  Is because the model was trained with 224x224 images the modal may not be able to label larger images?

**Comment by zmughal** [comment]
Yes, that is the specific input size that the model used in [the example](https://nbviewer.org/github/EntropyOrg/perl-AI-TensorFlow-Libtensorflow/blob/master/notebook/InferenceUsingTFHubMobileNetV2Model.ipynb) takes and how it was trained.

Each model has something called a signature that tells you the dimensions and type it takes for input and gives for output. Currently I recommend using the `saved_model_cli` script to help work that out. The specifics of how the input and output are encoded into the `TFTensor` varies from model to model, so you have to read the model documentation to find out what is expected.
